frontend/src/metabase-types/api/dataset.ts

```diff
@@ -43,14 +43,22 @@ export interface ResultsMetadata {
   columns: DatasetColumn[];
 }
 
+export type Insight = {
+  col: string;
+  unit: DatetimeUnit;
+  [key: string]: any;
+};
```

Comment:
WDYT about either dropping the `any` part or extending the type to match reality? Example:

<img width="469" alt="CleanShot 2023-12-13 at 17 20 23@2x" src="https://github.com/metabase/metabase/assets/17258145/ddb313b0-ae54-4bba-8590-dcaf6836960b">

Reply:
I am going to drop it and leave it as an exercise for someone who ends up working on code that will use those types, since they will have better knowledge of what those types look like and how they are used.